doc: fix setter of `event.cancelBubble` (#45986)
nodejs/node: doc/api/events.md

This is not used in Node.js and is provided purely for completeness.

> #### `event.cancelBubble(value)`

Reviewer: ```suggestion
#### `event.cancelBubble`
```

<!-- YAML
added: v14.5.0
-->

> * {boolean}

Reviewer: Sorry, my previous suggestion was incomplete
```suggestion
* Type: {boolean}
```

Author: I also missed it. Thank you for your help.

> * `value` {boolean}

Reviewer: ```suggestion
* {boolean}
```

> * `value` {boolean}

Reviewer: ```suggestion
* {boolean}
```

> Alias for `event.stopPropagation()` if `value` is `true`. This is not used

Reviewer: ```suggestion
Alias for `event.stopPropagation()` if set to `true`. This is not used
```
